### PR TITLE
Internationalisation

### PR DESCRIPTION
**Pull request by wszarmach**
First phase of internationalisation

### CODE REVIEW

File: schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-subscribers.js [hunk 1]
```diff
@@ -30,7 +33,8 @@
         }
 
         function fail(r) {
-            vm.alert('danger', 'Failed: ' + r.statusText);
+            var translation = $translate('FAILED');
+            vm.alert('danger', translation + ': ' + r.statusText);
         }
```
Comment (krico) [inline schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-subscribers.js]: Are we going to translate the "statusText" on the backend?  One option would be to say `$trasnlate(r.statusText)` but then any other client of the API would need to implement translations.

File: schedule/schedule-appengine/src/main/client/jasify/authenticate/sign-in.js [hunk 2]
```diff
@@ -18,6 +18,12 @@
             vm.user.email = BrowserData.getRememberUser();
         }
 
+        $rootScope.$on(AUTH_EVENTS.loginFailed, loginFailed);
+
+        function loginFailed() {
+            jasDialogs.resultError("There was an error with your E-Mail/Password combination. Please try again.");
+        }
```
Comment (krico) [inline schedule/schedule-appengine/src/main/client/jasify/authenticate/sign-in.js]: Oops... non-translated text here...

File: schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/activity-packages.selection.html [hunk 3]
```diff
@@ -30,22 +30,22 @@ <h2 class="lvh-label">{{activityPackage.name}}</h2>
...
                                       class="label label-warning">
-                                    {{vm.activityPackageSelection[activityPackage.id].length || 0}}/{{activityPackage.itemCount}} Activities selected.</span>
+                                    {{vm.activityPackageSelection[activityPackage.id].length || 0}}/{{activityPackage.itemCount}} {{ 'ACTIVITEIS_SELECTED' | translate }}</span>
                                 <span ng-show="vm.packageSelectionComplete(activityPackage)"
                                       class="label label-info">
-                                    Package selection completed.</span>
+                                    {{ 'SELECTION_COMPLETED' | translate }}</span>
                                 <span ng-show="vm.packageSelectionTooBig(activityPackage)"
                                       class="label label-info">
                                     You selected to many activities. Exactly {{activityPackage.itemCount}} activities must be selected.</span>
```
Comment (krico) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/activity-packages.selection.html]: Missing translation `You selected to many...`

Comment (pauljanes) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/activity-packages.selection.html]: It should be "too many" by the way, not "to many"

Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/jasify/booking-via-jasify/activity-packages.selection.html]: This one is pending. I had a problem with dynamic values and formatting and have aske Pawel for help. Have raised issue for it so I dont forget, #608

File: schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-subscribers.js (hunk 1 above)
Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/jasify/admin/activity/admin-subscribers.js]: We could use error codes and only display those or use them to find the correct translation on the front end.

File: schedule/schedule-appengine/src/main/client/jasify/authenticate/sign-in.js (hunk 2 above)
Comment (wszarmach) [inline schedule/schedule-appengine/src/main/client/jasify/authenticate/sign-in.js]: Good spot

**Pull request merged**